docs/index.md

```diff
@@ -1,7 +1,12 @@
-# Cirrus CI
+<h2 class="text-center">Make your development cycle <b>fast</b>, <b>efficient</b>, and <b>secure</b> with modern cloud technologies and Cirrus CI.</h2> 
 
-Cirrus CI makes your development cycle **fast**, **efficient**, and **secure** by leveraging modern cloud technologies. 
-Cirrus CI scales with your team and makes shipping software faster and cheaper.
+<img src="/assets/images/screenshots/typical-build.png" />
+
+<a href="/features">
+  <button class="btn center">
+    <b>See All The Features</b>
```

Comment:
```suggestion
    <b>See all the Features</b>
```
typically transition words are lowercase

Reply:
I think inside buttons usually every word starts with a capital letter. 🤔

Reply:
Changed to `Explore the Features`